packages/js/src/lib/initialize.ts

```diff
@@ -97,8 +105,15 @@ export const initialize = async (
       userId: c.userId,
     });
 
+    // if userId and attributes are available, set them
+    if (c.userId && c.attributes) {
```

Comment:
here we then also need to check if these attributes are already set. We either put this in the function to update the attributes (as it is now with setPersonAttribute) or only do this for the else case here